app/src/main/kotlin/no/nav/k9punsj/omsorgspengermidlertidigalene/OmsorgspengerMidlertidigAleneSøknadDto.kt

```diff
@@ -11,6 +11,13 @@ import no.nav.k9punsj.objectMapper
 import java.time.LocalDate
 import java.time.LocalTime
 
+data class NyOmsMASøknad(
+    val norskIdent: String,
+    val journalpostId: String,
+    val annenPart: String? = null,
+    val barn: List<OmsorgspengerMidlertidigAleneSøknadDto.BarnDto>
```

Comment:
Denne kunde extraheres til felles DTO, detsamme i andre ytelser med bl.a. PSB.

Reply:
Det er greit. Men tar det isåfall som egen commit/PR. 
Den PR-en fokuserer kun på fix for barn ved oppretting av ny søknad for oms-ma.